contracts/ForkDelta.sol

```diff
@@ -347,4 +369,79 @@ contract ForkDelta {
     orderFills[msg.sender][hash] = amountGet;
     emit Cancel(tokenGet, amountGet, tokenGive, amountGive, expires, nonce, msg.sender, v, r, s);
   }
+  
+  ////////////////////////////////////////////////////////////////////////////////
+  // Contract Versioning / Migration
+  ////////////////////////////////////////////////////////////////////////////////
+  
+  /**
+  * User triggered function to migrate funds into a new contract to ease updates.
+  * Emits a FundsMigrated event.
+  * @param address Contract address of the new contract we are migrating funds to
+  * @param address[] Array of token addresses that we will be migrating to the new contract
+  */
+  function migrateFunds(address newContract, address[] tokens_) {
```

Comment:
My reasoning behind passing in the contract was because:
- It simplifies the logic in the contract
- It allows us / users to migrate funds to any ForkDelta contract (or really any contract that makes use of our protocol)

I am open to suggestions, but it just seems like one less problem if we can select the specific contract to "upgrade" to especially if we have multiple. I understand the looped logic did that, but only to an extent and it was somewhat cumbersome.